### DISCUSSION TITLE
editor's Copilot extension, version "copilot/1.84.51", is out of date.

### DISCUSSION THREAD

**Discussion by Kuznetsov-100-Rads-Bar** [community/community#64000] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello I have a problem with my copilot here is the message:
"Copilot extension, version copilot /1.84.51 is out of date"
a newer version into access this service.

right now my copilot no longer works?

I uninstalled my copilot and I reinstalled copilot
but it still does not work?

my version Visual studio code : Version: 1.70.0 (user setup)
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Windows_NT x64

[INFO] [copilotIgnore] [2023-08-17T01:15:04.469Z] inactive
[INFO] [default] [2023-08-17T01:19:52.422Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2023-08-17T01:19:52.498Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 76 ms
[INFO] [fetch] [2023-08-17T01:19:52.501Z] Your editor's Copilot extension, version "copilot/1.84.51", is out of date. Please install a newer version to access this service.

please how do I follow the procedure
to get the new version of my copilot to have access to this service?

THANKS

**Comment by davecheney** [comment, marked as answer]
Control Shift P, type extensions and choose "Check for extension updates"

![Screenshot 2023-08-17 at 13 13 31](https://github.com/community/community/assets/7171/337a587d-ca3f-4351-93b6-0f9b8d8eba41)

You will also have to update your version of VSCode to 1.75 or later.

**Comment by Kuznetsov-100-Rads-Bar** [comment]
Thank you for your quick response!

Yes I also updated my VSC Version: 1.80.1 (system setup)
  
I think it should work again

Anyway thank you very much
